extensions/amp-tiktok/amp-tiktok.md

```diff
@@ -0,0 +1,159 @@
+
+## Attributes
+
+### `width`
+
+The `width` attribute represents the width of the `amp-tiktok` element.
+However the `iframe` child element will be set to '325px'.
+
+### `height`
+
+The `height` refers to the size of the `amp-tiktok` element. However the Tiktok player is '575px' tall. If the `height` that is set is less than '575px' then the element will resize to fit that new height. This will cause layout shift.
+
+In order to prevent layout shifting, the `height` should set to be at least '575px'.
+
+### `data-src`
+
+The `data-src` attribute can contain one of two values. Either it can contain the video-id or the full source url for the TikTok.
+
+Example with video-id
+
+```html
+ <amp-tiktok width="325" height="575" data-src="6718335390845095173">
+```
+
+Example with source url:
+
+```html
+    <amp-tiktok
+      width="325 "
+      height="575"
+      data-src="https://www.tiktok.com/@scout2015/video/6718335390845095173"
+    >
+```
+
+In liu of the `data-src` attribute, `amp-tiktok` also allows the use of the TikTok generated embed code to display.
+
+To use this method copy the blockquote from the TikTok Embed code:
+
+1. Go to the desktop page of the TikTok you wish to embed
+2. Click the embed link icon (`</>`)
+3. Copy the embed code and add it as a child element of the `amp-tiktok` element.
+4. Add the `placeholder` attribute to the `blockquote`.
+5. Remove the `style` attribute from the `blockquote`.
+
+```html
+<amp-tiktok width="300" height="800">
+      <blockquote
+        placeholder
+        class="tiktok-embed"
+        cite="https://www.tiktok.com/@scout2015/video/6718335390845095173"
+        data-video-id="6718335390845095173"
+        style="max-width: 605px; min-width: 325px"
+      >
+        <section>
+          <a
+            target="_blank"
+            title="@scout2015"
+            href="https://www.tiktok.com/@scout2015"
+            >@scout2015</a
+          >
+          <p>
+            Scramble up ur name & I’ll try to guess it😍❤️
+            <a
+              title="foryoupage"
+              target="_blank"
+              href="https://www.tiktok.com/tag/foryoupage"
+              >#foryoupage</a
+            >
+            <a
+              title="petsoftiktok"
+              target="_blank"
+              href="https://www.tiktok.com/tag/petsoftiktok"
+              >#petsoftiktok</a
+            >
+            <a
+              title="aesthetic"
+              target="_blank"
+              href="https://www.tiktok.com/tag/aesthetic"
+              >#aesthetic</a
+            >
+          </p>
+          <a
+            target="_blank"
+            title="♬ original sound - tiff"
+            href="https://www.tiktok.com/music/original-sound-6689804660171082501"
+            >♬ original sound - tiff</a
+          >
+        </section>
+      </blockquote>
+    </amp-tiktok>
+```
+
+## Accessibility (optional)
+
+The `aria-title` will be set to 'TikTok'.
```

Comment:
I don't think this should be the case. It should instead cascade: (also `aria`**`-label`**, not `-title`)

1. If `<amp-tiktok aria-label>` is provided, it's propagated to the iframe: `<iframe aria-label>`
2. If we can get metada (per full URL), it should be `TikTok: [TIKTOK DESCRIPTION]`
3. Otherwise, just set to `TikTok`

Reply:
The PR which adds the metadata functionality is separate from this one, should I add the second cascade step in that PR for clarity and simply describe scenarios 1 and 3 here?

Reply:
Yep, let's skip 2 on this PR and propagate/document aria-label here.